### PR TITLE
CC create cluster workflow

### LINKED ISSUE #4944: CC: Create a cluster in the managed console  V1
Background: https://airtable.com/tblD3oZPLJgGhCmch/viwchkdQkRUw6hZ3z/recnbQn3CTHOOt4xQ?blocks=hide
PM: @kannanlakshmi 

To-do:

- [ ] Incorporate Tim V.'s feedback: https://docs.google.com/document/d/1p4ZSR8Ris2ak0-PlEW1JGSlkHwfSXly9Innn79FqkFs/edit
- [ ] Ask CC FE team to add doc links from the console to the price comparison doc and hardware guidance doc

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #4944 #5643 #5474 #5737

### CODE REVIEW

**Comment by Amruta-Ranade** [comment]
___
*[v19.2/cockroachcloud-create-your-cluster.md, line 45 at r6](https://reviewable.io/reviews/cockroachdb/docs/5680#-LswrykT1O8mmPbVrlEq:-LswwkBp3TtV2fIltzUr:b-c5g0pj) ([raw file](https://github.com/cockroachdb/docs/blob/a3cc5e8af877842209e84ecafca800382f782cbd/v19.2/cockroachcloud-create-your-cluster.md#L45)):*
<details><summary><i>Previously, carloruiz wrote…</i></summary><blockquote>

For developers with not a lot of DB admin experience (like me) , this section is still daunting, particularly buffer and compression.  It would be really, really nice to explicitly note either rules of thumb (e.g. 5% buffer) for each section or even better, to include links to configurations for some example applications. I know this is going above and beyond what I've seen other providers (GCP/AWS) do but in my experience, I am super thankful when I encounter a product that goes above and beyond to welcome young developers. 
</blockquote></details>

I completely agree. I love the suggestion of having rule-of-thumb numbers for each parameter (although I don't know if we have that info -- will dig into it) and pointing to sample apps.

I am currently constrained by my own understanding of how the selection process works. In this first version of this doc, I wanted to at least point out the things they should be considering (since they aren't obvious at all to a new user). I hope I will be able to learn more about the selection process and improve this doc post-beta.

**Comment by Amruta-Ranade** [comment]
Note to self: add vpc peering as a reason for selecting cloud.

**Comment by Amruta-Ranade** [comment]
From Ben's Office Hours discussion: The number of nodes depends on performance, not the hardware config selection. Need to add example on number of nodes selection. (Use the TPC-C example Ben shared)

File: v19.2/cockroachcloud-create-your-cluster.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Compression | The percentage of savings you can expect to achieve with compression. With Snappy, the default algorithm, we typically see about a 40% savings on raw data size.  
+
+To change the hardware configuration after the cluster is created, you will have to contact us.
+
+The choice of hardware per node determines the throughput and performance characteristics of your cluster.
+
+The following table gives the performance characteristics for YCSB and TPC-C workloads per hardware configuration:
+
+<Performance characteristics table>
+<Also include IOPS numbers for each config>
```
Comment (kannanlakshmi) [inline v19.2/cockroachcloud-create-your-cluster.md]: I'd either not include iops number or if we do include IOPS number specify what that means in GCP v AWS. In GCP we dont get to pick IOPS. In AWS, these are provisioned IOPS.

File: v19.2/cockroachcloud-create-your-cluster.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+## Step 4. Select the hardware configuration
+
+While selecting the hardware configuration, consider the following factors:
+
+Factor | Description
+----------|------------
+Capacity | Total raw data size you expect to store without replication.
+Replication | The default replication factor for a CockroachCloud cluster is 3.
+Buffer | Additional buffer (overhead data, accounting for data growth, etc.).
+Compression | The percentage of savings you can expect to achieve with compression. With Snappy, the default algorithm, we typically see about a 40% savings on raw data size.
```
[the remark this answers is not in the document]
Comment (Amruta-Ranade) [reply v19.2/cockroachcloud-create-your-cluster.md]: Good point. Reworded to "CockroachDB's default compression algorithm"

File: v19.2/cockroachcloud-create-your-cluster.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+For optimal performance, select the cloud provider region in which you are running your application. For example, if your application is deployed in GCP's `us-east1` region, select `us-east1` for your CockroachCloud cluster.
+
+### Select number of nodes
+
+- For single-region application development and testing, you may create a one-node cluster.
+- For single-region production deployments, we recommend a minimum of three nodes. The number of nodes also depends on your storage capacity requirements. See Example for further guidance.
+- For multi-region deployments, [contact us](mailto:sales@cockroachlabs.com).
+
+As of now, you can add a maximum of 24 nodes to your cluster.
```
[the remark this answers is not in the document]
Comment (Amruta-Ranade) [reply v19.2/cockroachcloud-create-your-cluster.md]: Done.

File: v19.2/cockroachcloud-create-your-cluster.md (hunk 1 above)
Comment (Amruta-Ranade) [reply v19.2/cockroachcloud-create-your-cluster.md]: Punting this to the hardware guidance/perf tests PR.

**Comment by Amruta-Ranade** [comment]
Update: I am breaking this PR into two parts and will handle the perf test-related content and examples in a follow-up PR.

**Pull request merged**